openreview/conference/templates/programchairWebfield.js

```diff
@@ -1748,7 +1748,7 @@ var displayRejectedWithdrawnPaperStatusTable = function () {
       paperNumbers.push(note.number);
 
       var numberHtml = '<strong class="note-number">' + note.number + '</strong>';
-      var summaryHtml = Handlebars.templates.noteSummary(note.details.original || note);
+      var summaryHtml = Handlebars.templates.noteSummary(note.details ? note.details.original || note : note);
```

Comment:
don't you need to check if `original` exists too? We can use the details for single blind submissions...

Reply:
I see the || now!, thanks